tensorflow/python/tools/optimize_for_inference_lib.py

```diff
@@ -361,7 +361,15 @@ def fuse_resize_and_conv(input_graph_def):
     else:
       raise ValueError("Duplicate node names detected for ", node.name)
 
-  nodes_to_skip = {}
+  node_reference_count = {}
```

Comment:
if you use a collections.defaultdict  then you don't need to check for existence in the loop - just do  node_reference_count[stripped_name] += 1